common/src/main/java/com/gentics/mesh/verticle/BinaryCheckVerticle.java

```diff
@@ -180,11 +180,11 @@ public void checkBinary(BinaryCheckContext ctx) {
 		OkHttpClient client = new OkHttpClient.Builder().build();
 		Request request = new Request.Builder()
 			.url(ctx.checkServiceUrl)
-			.post(RequestBody.create(MediaType.parse("application/json"), checkRequest.toJson()))
+			.post(RequestBody.create(MediaType.parse("application/json"), checkRequest.toJson(true)))
 			.build();
 
 		if (log.isTraceEnabled()) {
-			log.trace("Performing binary check: POST {}\n{}", ctx.checkServiceUrl, new JsonObject(checkRequest.toJson()).encodePrettily());
+			log.trace("Performing binary check: POST {}\n{}", ctx.checkServiceUrl, new JsonObject(checkRequest.toJson(false)).encodePrettily());
```

Comment:
All the human readable JSONs (logs, debuginfos) are forcefully excluded from minification.